reviewresponse.cls

```diff
@@ -105,6 +109,10 @@
 	\textbf{Response:} #1\par
 }{\vspace{4em plus 0.2em minus 1.5em}}
 
+\newenvironment{concludingresponse}[1][]{%
+	\begin{tcolorbox}[attach title to upper, title={Concluding Response #1}, after title={.\enskip}, fonttitle={\bfseries}, coltitle={colorconcludingresponsefg}, colback={colorconcludingresponsebg}, colframe={colorconcludingresponseframe},]
```

Comment:
You can hardcode the `breakable` option into the environment. (Similar to the `changes` environment)
As far as I know, this does not have any disadvantages, unless you never want this type of box to be spanning multiple pages. Since this is probably the rarer case, one could manually adjust spacing/page breaks in this case.

```suggestion
	\begin{tcolorbox}[attach title to upper, breakable, title={Concluding Response #1}, after title={.\enskip}, fonttitle={\bfseries}, coltitle={colorconcludingresponsefg}, colback={colorconcludingresponsebg}, colframe={colorconcludingresponseframe},]
```